app/views/govuk_publishing_components/components/docs/action_link.yml

```diff
@@ -49,6 +49,27 @@ examples:
       light_text: true
     context:
       dark_background: true
+  mobile_subtext:
+    description: Lock the subtext beneath the main text, so the component with subtext always appears as it does on mobile.
+    data:
+      text: "Try not to panic now"
+      href: '/page'
+      subtext: "The thing that is happening is pretty scary but we'll get through it"
+      mobile_subtext: true
+  with_link_on_subtext:
+    description: The subtext can also be a link if required. The link on the main text is optional.
+    data:
+      text: "Remain calm"
```

Comment:
Here and on a few more instances, the double quotes are redundant.

Reply:
Thanks, have addressed this.